docs/catalog-restrictions.md

```diff
@@ -62,35 +62,39 @@ subset of properties on service class and service plan resources. The following
 
 `ClusterServiceClass` allowed property names:
 
-| Property Key    | Description    |
-| name |  This key will match the ClusterServiceClass.Name property |
-| spec.externalName | This key will match the ClusterServiceClass.Spec.ExternalName property |
-| spec.externalID | This key will match the ClusterServiceClass.Spec.ExternalID property |
+| Property Key    | Description    
```

Comment:
sorry to be picky, but trailing too since all other tables in or docs have those.

Reply:
Done.